src/pages/SwapV2/index.tsx

```diff
@@ -624,6 +626,7 @@ export default function Swap({ history }: RouteComponentProps) {
 
   useEffect(() => {
     checkAutoSelectTokenFromUrl()
+    initialTotalTokenDefault.current = Object.keys(defaultTokens).length
```

Comment:
it can be moved outside of this effect

Reply:
What if @nguyenhoaidanh indent for this value not update realtime but only after some deps updated? In this case it is only first time initially (no deps)

Reply:
Should have some explain from @nguyenhoaidanh

Reply:
Ah yeah, my bad. An explanation here is good

Reply:
yes, Nam is correct. I named it `initial`TotalTokenDefault.     I just want to get init total count of default token.